tests/e2e/taqueria-plugin-flextesa.spec.ts

```diff
@@ -0,0 +1,240 @@
+        try {
+            // 1. Run list accounts command on sandbox that is not running and verify result
+            const stdoutSandboxIsNotRunning = execSync("taq list accounts local", {cwd: `./${taqueriaProjectPath}`}).toString().trim();
+            expect(stdoutSandboxIsNotRunning).toEqual("The local sandbox is not running.");
+
+        } catch(error) {
+            throw new Error (`error: ${error}`);
+        }
+
+    });
+
+    test('Verify that taqueria flextesa plugin will return "he local sandbox was not running." if user tries to call stop on sandbox that is not running', async () => {
+        try {
+            // 1. Run stop sandbox local on sandbox that is not running and verify result
+            const stdoutSandboxIsNotRunning = execSync("taq stop sandbox local", {cwd: `./${taqueriaProjectPath}`}).toString().trim();
+            expect(stdoutSandboxIsNotRunning).toEqual("The local sandbox was not running.");
+
+        } catch(error) {
+            throw new Error (`error: ${error}`);
+        }
+
+    });
+
+
+    // TODO: Currently it cannot be done until this issue has been resolved
+    // https://github.com/ecadlabs/taqueria/issues/243
+    test.skip('Verify that taqueria flextesa plugin can start and stop a sandbox with custom name', async () => {
+        try {
+            // Setting up docker container name
+            dockerName = "test"
+
+            // 1. Updating config to add test sandbox
+            execSync(`cp e2e/data/config-flextesa-test-sandbox.json ${taqueriaProjectPath}/.taq/config.json`);
+
+            // 2. Run sandbox start command with customer name
+            const stdoutStart = execSync(`taq start sandbox ${dockerName}`, {cwd: `./${taqueriaProjectPath}`}).toString().trim();
+
+            // 3. Verify that sandbox has been started and taqueria returns proper message into console
+            expect(stdoutStart).toEqual("Started test.");
+
+            // 4. Run list accounts command and verify that it returns list of default accounts
+            await waitForExpect(() => {
+                const stdoutList = execSync(`taq list accounts ${dockerName}`, {cwd: `./${taqueriaProjectPath}`}).toString().trim();
+                expect(stdoutList).toContain("ken");
+                expect(stdoutList).not.toContain("tran");
+            });
+
+            // 5.  Run stop command and verify the output
+            const stdoutStop = execSync(`taq stop sandbox ${dockerName}`, {cwd: `./${taqueriaProjectPath}`}).toString().trim();
+
+            // 6. Verify that taqueria returns proper message into console
+            expect(stdoutStop).toEqual("Stopped test.");
+
+        } catch(error) {
+            throw new Error (`error: ${error}`);
+        }
+
+    });
+
+
+    // TODO: Currently it cannot be done until this issue has been resolved
```

Comment:
Ivan, do you mind also creating an issue for finishing this test case?  In the description, you can note that the issue is blocked by #243.

When creating the issue, there are some fields on the left-side of the issue entry screen. If you could set "Project" to "Taqueria", assign to yourself, add the "blocked" label, and then move to the "Blocked" column in the project board, that would be great. Let me know if you have any questions.

Finally, if you could add a link to that new issue in your TODO comment it would be much appreciated. We then won't forgot to come back here when #243 is resolved.